UnchainedLauncher.GUI/Views/MainWindow.xaml

```diff
@@ -1,15 +1,36 @@
 ﻿<Window
         xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
         xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-        xmlns:av="http://schemas.microsoft.com/expression/blend/2008" 
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
         xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-        x:Name="Unchained_Launcher" 
-        mc:Ignorable="av" 
+        xmlns:local="clr-namespace:UnchainedLauncher.GUI.Views"
+        xmlns:vms="clr-namespace:UnchainedLauncher.GUI.ViewModels"
+        mc:Ignorable="d" 
         x:Class="UnchainedLauncher.GUI.Views.MainWindow"
         Title="Unchained Launcher" Height="280" Width="650"
     >
+    
+    <d:DesignProperties.DataContext>
+        <vms:MainWindowViewModel />
+    </d:DesignProperties.DataContext>
 
     <Window.Resources>
+        <DataTemplate DataType="{x:Type vms:LauncherViewModel}">
+            <local:LauncherView />
+        </DataTemplate>
+        <DataTemplate DataType="{x:Type vms:ModListViewModel}">
+            <local:ModListView />
+        </DataTemplate>
+        <DataTemplate DataType="{x:Type vms:ServerLauncherViewModel}">
+            <local:ServerLauncherView />
+        </DataTemplate>
+        <DataTemplate DataType="{x:Type vms:SettingsViewModel}">
+            <local:SettingsView />
+        </DataTemplate>
+        <DataTemplate DataType="{x:Type vms:ServersViewModel}">
+            <local:ServersView />
+        </DataTemplate>
```

Comment:
Each data template binds a ViewModel to a View. Later, down below, you'll see `ContentControl`s that are bound to a ViewModel.  WPF checks the type of the ViewModel, then the corresponding View from the DataTemplate mapping.